From 6a16c4c268962fd09a52e4e78116aaa3bb1c47d5 Mon Sep 17 00:00:00 2001
From: Alan Coopersmith <alan.coopersmith@oracle.com>
Date: Wed, 1 Jun 2011 21:42:07 -0700
Subject: [PATCH 5/7] "privates.h", line 198: warning: void function cannot
 return value

Providing an argument to return in a function with void return type
is not allowed by the C standard, and makes the Sun compilers unhappy.
(They actually flag it as an error, unless using a new enough version
 to be able to downgrade it to a warning with "-features=extensions".)

Signed-off-by: Alan Coopersmith <alan.coopersmith@oracle.com>
Reviewed-by: Jeremy Huddleston <jeremyhu@apple.com>
Reviewed-by: Daniel Stone <daniel@fooishbar.org>
Reviewed-by: Cyril Brulebois <kibi@debian.org>
(cherry picked from commit bc04065b5ce277f3ac3491ff221a60ef3c7605cf)
---
 include/privates.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git xorg-server-1.10.2/include/privates.h xorg-server-1.10.2/include/privates.h
index 7ef2cb7..2b0040c 100644
--- xorg-server-1.10.2/include/privates.h
+++ xorg-server-1.10.2/include/privates.h
@@ -195,7 +195,7 @@ dixGetScreenPrivate(PrivatePtr *privates, const DevScreenPrivateKey key, ScreenP
 static inline void
 dixSetScreenPrivate(PrivatePtr *privates, const DevScreenPrivateKey key, ScreenPtr pScreen, pointer val)
 {
-    return dixSetPrivate(privates, _dixGetScreenPrivateKey(key, pScreen), val);
+    dixSetPrivate(privates, _dixGetScreenPrivateKey(key, pScreen), val);
 }
 
 static inline pointer
-- 
1.7.5.2

